dashboard/15-final/app/ui/dashboard/sidenav.tsx

```diff
@@ -5,20 +5,23 @@ import LogOutButton from './log-out-button';
 
 export default function SideNav() {
   return (
-    <div className="flex h-full flex-col justify-between border-r px-2 py-4">
-      <div>
-        <Link href="/">
-          <Image
-            src="/logo.png"
-            alt="The Next.js Symbol, a white N inside a black circle"
-            className="mb-4 ml-1"
-            width={32}
-            height={32}
-          />
-        </Link>
+    <div className="flex h-full flex-col border-r px-2 py-4">
+      <Link
+        className="mb-2 flex h-20 items-end rounded-md bg-blue-600 p-4 md:h-40"
+        href="/"
+      >
+        <Image
+          src="/acme-logo.svg"
+          alt="The Next.js Symbol, a white N inside a black circle"
+          width={100}
+          height={100}
```

Comment:
Let's use the equivalent ratios. Maybe we can also re-upload it to remove the extra space at the top and bottom. 

```suggestion
          width={139}
          height={32}
```

Reply:
It may be simpler to do the SVG in JSX because of: https://nextjs.org/docs/app/api-reference/components/image#dangerouslyallowsvg